fluent-langneg/src/headers.js

```diff
@@ -0,0 +1,8 @@
+
+export default function parseAcceptedLanguagesHeaderIntoArray(string = '') {
+  if (typeof string !== 'string') throw new TypeError('Argument must be a string');
+  const tokens = string.split(',').map(t => t.trim());
+  return tokens.filter(t => t !== '').map(t => {
+    return t.split(';')[0];
```

Comment:
yes, this is on purpose. We don't act upon the q-value at all. The order is the only indication for our fallback, there's no increase/decrease in fallback based on the q-value.